Plugins/Reqnroll.NUnit.Generator.ReqnrollPlugin/build/NUnit.AssemblyHooks.template.cs

```diff
@@ -6,10 +6,9 @@
 using global::System.Runtime.CompilerServices;
 using System.Threading.Tasks;
 
-[assembly: NUnit.Framework.FixtureLifeCycle(NUnit.Framework.LifeCycle.InstancePerTestCase)]
-
 [GeneratedCode("Reqnroll", "REQNROLL_VERSION")]
 [global::NUnit.Framework.SetUpFixture]
+[global::NUnit.Framework.FixtureLifeCycle(global::NUnit.Framework.LifeCycle.InstancePerTestCase)]
```

Comment:
Do we need to specify the attribute for a `SetUpFixture'?

SetUpFixture is global and should only have one instance.

It probably won't hurt either way, just wanted to ask.

Reply:
Seems to be working also without it... Removed.